server/views/subscription/web-manage.mjml.hbs

```diff
@@ -6,9 +6,6 @@
         <mj-text>
 {{> subscription_manage_form}}<!-- don't indent me! -->
         </mj-text>
-        <mj-button mj-class="button" href="#submit">
-            Update Profile
-        </mj-button>
```

Comment:
Uses the submit button in `server/views/subscription/partials/subscription-manage-form.hbs`